### ISSUE TITLE
left and right chevrons in scrolling tabs should not be in the tab order when not visible

### ISSUE THREAD

**Issue by anthonytordillos** [PolymerElements/paper-tabs#61]
You can repro this on the demo page. Make the viewport large enough that the right and left chevrons are not visible. Tab through the tabs, and you'll notice that the right chevron still receives focus.

**Comment by tjsavage** [comment]
I'm able to repro funkiness with the tab order as well
